shared/build.gradle.kts

```diff
@@ -60,7 +62,7 @@ kotlin {
         val iosX64Main by getting
         val iosArm64Main by getting
         val iosSimulatorArm64Main by getting
-        val iosMain by creating {
+        val iosMain by getting {
             dependsOn(commonMain)
             iosX64Main.dependsOn(this)
```

Comment:
if we use the default hierarchy, then it is not required to have manual dependsOn edges

Reply:
Thank you